_posts/2018-01-04-notification-channels-android-o.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title:  "Notification Channels in Android Oreo"
+date:   2018-02-06 20:20:02
+description: Prepare your app for Android Oreo targets by implementing notification channels
+excerpt: "Prepare your app for Android Oreo targets by implementing notification channels"
+permalink: /posts/notification-channels-android-o
+tags: [android oreo, tutorial, notification channels, notification categories]
+---
+
+
+In (not-so) recent news, Google is going to be enforcing that apps are targetted for SDK 26 (Android Oreo) by late 2018, in order to ensure security and improved performance in apps. You can read more about the announcement [here](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+
+Now, it's pretty easy to target API 26, but there's no gurantee that your app will function as it used to by just changing the `targetSdkVersion` value in your `build.gradle` file.
+
+This post looks specifically at one of the new things introduced in Android Oreo, and how targetting API 26 will affect your app, and notifications - notification channels (or categories).
+
+TL;DR
+-----
+
+Your app will stop sending notifications to the users if you target API 26 and you don't implement notification channels.
+
+What are Notification Channels?
+-----
+
+In Android O, notification channels are somewhat like groups or categories of notifications - say you're building a social networking app, the channels can be "activity" - like likes, or comments on your posts, "messages", etc. In fact, in the settings of your app, the user sees channels as "categories" as seen in the screenshot below.
```

Comment:
I think _the channels can be "activity" - like likes, or comments on your posts, "messages", etc._ is confusing

_the channels can be "activity" - likes, comments on your posts, messages, etc._

Reply:
Well, yea, true. Thanks! Fixed 🔒